### PR TITLE
feat: Add leaderboard and BattleTV

### PR DESCRIPTION
**Pull request by jaiakash**
- Leaderboard
- Battle TV
Screenshot from 2022-01-28 03-14-52
Screenshot from 2022-01-28 03-15-05

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -26,11 +26,14 @@
...
     "@reduxjs/toolkit": "^1.7.1",
+    "bootstrap": "^5.1.3",
     "lit": "^2.1.1",
     "phaser": "^3.55.2",
     "react": "^17.0.2",
+    "react-bootstrap": "^2.1.1",
     "react-dom": "^17.0.2",
-    "react-router-dom": "6"
+    "react-router-dom": "6",
+    "styled-components": "^5.3.3"
```
Comment (PriyanshuDangi) [inline package.json:36]: Is this used anywhere?
otherwise remove it

File: src/components/BattleTV/BattleTV.tsx [hunk 2]
```diff
@@ -0,0 +1,83 @@
+import 'bootstrap/dist/css/bootstrap.css';
+import './styles.css';
```
Comment (PriyanshuDangi) [inline src/components/BattleTV/BattleTV.tsx:2]: We are using module.css

File: src/components/BattleTV/BattleTV.tsx [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+            <div className="username">{row.myusername}</div>
+            <div className="coinused">{row.mycoinused}</div>
+            <div className="destruction">{row.mydestruction}</div>
+            <div className="vs">VS</div>
+            <div className="destruction">{row.enemydestruction}</div>
+            <div className="coinused">{row.enemycoinused}</div>
+            <div className="username">{row.enemyusername}</div>
+            <div
+              className="pic"
+              style={{
```
Comment (PriyanshuDangi) [inline src/components/BattleTV/BattleTV.tsx:74]: Why inline styles when we have css file

File: src/components/Leaderboard/Leaderboard.tsx [hunk 4]
```diff
@@ -0,0 +1,107 @@
+import 'bootstrap/dist/css/bootstrap.css';
```
Comment (PriyanshuDangi) [inline src/components/Leaderboard/Leaderboard.tsx:1]: Better to import boostrap once in main.tsx

File: src/components/Leaderboard/Leaderboard.tsx [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+import './styles.css';
```
Comment (PriyanshuDangi) [inline src/components/Leaderboard/Leaderboard.tsx:2]: Again change to css modules

File: src/components/Leaderboard/Leaderboard.tsx [hunk 6]
```diff
@@ -0,0 +1,107 @@
...
+  createData(8, 'Akash8', 122, 1, 1, 1, 'fightakash'),
+];
+
+export default function Leaderboard(): JSX.Element {
+  return (
+    <div className="body">
+      <div className="header">
+        <img
+          className="header__icon"
+          src="https://user-images.githubusercontent.com/23297041/55285200-a24e9b00-538f-11e9-8990-d49a162824d1.png"
```
Comment (PriyanshuDangi) [inline src/components/Leaderboard/Leaderboard.tsx:30]: better download this image and then use
above also

File: src/components/Leaderboard/Leaderboard.tsx [hunk 7]
```diff
@@ -0,0 +1,107 @@
...
+        </h1>
+      </div>
+      <div className="center">
+        <div className="top3">
+          <div className="two item">
+            <div className="pos">2</div>
+            <div
+              className="pic"
+              style={{
+                backgroundImage: `url("https://randomuser.me/api/portraits/men/34.jpg")`,
```
Comment (PriyanshuDangi) [inline src/components/Leaderboard/Leaderboard.tsx:43]: don't use inline styles

**Comment by jaiakash** [comment]
Made the necessary changes.
1. Module css instead of directly using css.
2. Removed unused imports.
3. Bootstrap import from main.tsx file.
4. Other changes.

**Comment by PriyanshuDangi** [comment]
Can you pull rebase and fix these conflicts
Also, we need react-router-dom
It shouldn't be removed

**Pull request closed without merging**